Source/WebInspectorUI/UserInterface/Views/ResourceTreeElement.js

```diff
@@ -84,7 +84,19 @@ WI.ResourceTreeElement = class ResourceTreeElement extends WI.SourceCodeTreeElem
         if (comparisonResult !== 0)
             return comparisonResult;
 
-        // Compare by title when the subtitles are the same.
+        // Compare by title without file extensions when the subtitles are the same.
+        function filenameWithoutExtention (resourceTitle) {
```

Comment:
Style: No space between function name and opening paren.

Reply:
Nit: s/extention/extension here and throughout below.

Reply:
Addressed in 673a204c276c6ecb5d1c3dc8d0df42c54c1b0b67.